miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/storage/MiniAppSecureStorage.kt

```diff
@@ -232,13 +231,13 @@ internal class MiniAppSecureStorage(private val activity: Activity) {
                     if (storedItems.containsKey(key)) {
                         onSuccess(storedItems[key] ?: "")
                     } else {
-                        onFailed(MiniAppSecureStorageError.secureStorageIOError)
+                        onSuccess("null")
                     }
                 } ?: kotlin.run {
-                    onFailed(MiniAppSecureStorageError.secureStorageIOError)
+                    onSuccess("null")
                 }
             } else {
-                onFailed(MiniAppSecureStorageError.secureStorageUnavailableError)
+                onSuccess("null")
```

Comment:
When storage is not available we should return an error. But remember whenever mini-app is opened secure storage will be available always unless the device storage is full already.

Reply:
@rleojoseph  in our case thats the case we get into after clear storage, we delete the file, so when you try to get the item the file is not there so it will throw error and show the error currently showing.